<launch>
	<!-- Joy -->
	<node pkg="joy" type="joy_node" name="joy0" ns="joy0" respawn="true" output="screen">
		<param name="dev" value="/dev/input/js0"/>
	</node>

	<node pkg="joy" type="joy_node" name="joy1" ns="joy1" respawn="true" output="screen">
		<param name="dev" value="/dev/input/js1"/>
	</node>

	<node pkg="arm_control" type="arm_controlv3" name="arm_controlv3" respawn="true" output="screen"/>
</launch>
